certbot-compatibility-test/certbot_compatibility_test/test_driver.py

```diff
@@ -329,13 +335,13 @@ def setup_logging(args):
     root_logger.addHandler(handler)
 
 
-def setup_display():
+def setup_display() -> None:
     """"Prepares a display utility instance for the Certbot plugins """
     displayer = display_util.NoninteractiveDisplay(sys.stdout)
     display_obj.set_display(displayer)
 
 
-def main():
+def main() -> None:
```

Comment:
In this function, my language server shows `plugin` as type `Any`, which creates some further `Any`s.

Could we type `PLUGINS` as `Dict[str, Type[common.Proxy]]`, so the types in `main` are more precise?

Reply:
Fixed.